ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/services/ApiSubmissionService.scala

```diff
@@ -164,10 +196,32 @@ final class ApiSubmissionService private (
           commands.commands
         )
       submissionResult <- handleResult(res)
-    } yield submissionResult
+    } yield submissionResult).transform {
```

Comment:
Do you think it would make sense to put the function passed to transform as a private method and combine the two with `transform` in a separate place?